File: segundos 4 lenguajes/c#/ProyectoCsharp/JugadorNovato.cs
namespace ProyectoJugadores
{
    public class JugadorNovato
    {
        public string Nombre { get; set; }
        public int Nivel { get; set; }

        public JugadorNovato()
        {
            Nombre = "Novato";
            Nivel = 1;
        }

        public JugadorNovato(string nombre, int nivel)
        {
            Nombre = nombre;
            Nivel = nivel;
        }

        public static JugadorNovato Copy(JugadorNovato j)
        {
            return new JugadorNovato(j.Nombre, j.Nivel);
        }
    }
}

File: segundos 4 lenguajes/js/JugadorNovato.js
export class JugadorNovato {
    constructor(nombre = "Novato", nivel = 1, copia = null) {
        if (copia) {
            this.nombre = copia.nombre;
            this.nivel = copia.nivel;
        } else {
            this.nombre = nombre;
            this.nivel = nivel;
        }
    }
}
